apps/wallet/src/ui/app/components/WalletListSelect.tsx

```diff
@@ -113,6 +113,7 @@ export function WalletListSelect({
 				) : null
 			}
 			minimalPadding
+			boxShadow
```

Comment:
This component is used in other places as well and not sure if the shadow there is intended.
<img width="359" alt="Screenshot 2023-06-26 at 12 06 34" src="https://github.com/MystenLabs/sui/assets/10210143/30a3da4c-a52b-4ef0-89e6-f54b674f3fc3">